src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/MuTect2.java

```diff
@@ -607,47 +456,40 @@ public void apply(final AssemblyRegion region, final ReferenceContext referenceC
         //TODO: verify that vc.getStart() is the right second argument to use here
         final Collection<VariantContext> panelOfNormalsVC = featureContext.getValues(MTAC.normalPanelFeatureInput, vc.getStart());
 
-        //TODO: is this the correct criterion for filtering?
-        //TODO: Takutp points out that we're filtering based on the PoN even if the alleles are different!
-        //TODO: and what about genotypes?
+        //TODO: Takuto points out that we're filtering even if the alleles are different!  And what about genotypes?
```

Comment:
although it's true that the PON will filter out gemline events (where alleles would matter), if you have a normal it does a WAY better job at that.  Then then PON is primarily used as a "noise filter" (and has a huge effect) but in those cases we don't really trust the data other than "this is a noisy site... don't trust the data" and the alleles don't matter.  

Tumor-only calling is a different thing... and you probably want some of each of the above ;)